source/_integrations/flipr.markdown

```diff
@@ -0,0 +1,47 @@
+---
+title: Flipr
+description: Instructions on how to integrate your Flipr device within Home Assistant.
+ha_category:
+  - Sensor
+ha_release: 2021.2
+ha_iot_class: Cloud Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@cnico'
+ha_domain: flipr
+---
+
+[Flipr](https://www.goflipr.com) is a smart pool monitor that publishes data to the cloud via SigFox.
+Flipr sends data like ph, chlorine or temperature to a cloud server on a regular basis in order to monitor your pool.
+This integration gives you access to these information on Home Assistant in addition the vendor's smartphone application.
+
+There is currently support for the following information within Home Assistant:
+
+- Chlorine Level
+- pH
+- Water Temperature
+- Red OX Level
+- Last measure date
+- pH Status Indicator
+- Chlorine Status Indicator
+
+## Configuration
+
+You will need to use the standalone app for this device to register a username and password.
+
+- [Google](https://play.google.com/store/apps/details?id=com.goflipr.flipr)
+- [Apple](https://apps.apple.com/fr/app/flipr/id1225898851)
+
+Home Assistant offers Flipr integration through **Configuration** -> **Integrations**. Click the `+` sign and from the list of integrations select **Flipr**. Simply enter `username` and `password` when prompted. Your flipr ID is automatically determined and if not you can either enter it or select it through a list of values.
```

Comment:
You could use the following snippet here.

```yaml
{% include integrations/config_flow.md %}
```

Reply:
I do not understand what you mean here and what this include would do...
I corrected the documentation with changes we agreed with your review of the PR.

Reply:
This will include the following snippet: https://github.com/home-assistant/home-assistant.io/blob/current/source/_includes/integrations/config_flow.md.

Reply:
OK.
I prefer my version which is more specific and clearer.
If this is required, tell me I can change but I prefer not.